### PR TITLE
Use $lib/holocene instead of $holocene

### PR DESCRIPTION
**Pull request by Alex-Tideman**
## What was changed
Update references of $holocene to $lib/holocene if it's a holocene file.

## Why?
Fix importing of $holocene alias for npm package

### CODE REVIEW

**Review by rossedfort — APPROVED** [review]
Maybe we can dig into why it's not working another time, but I don't want to hold up merging the upgrade 👍

**Comment by Alex-Tideman** [comment]
For sure, I'm going to beta publish this first and see if it actually fixes the problem before merging

File: src/lib/holocene/accordion.svelte
Comment (laurakwhit) [inline src/lib/holocene/accordion.svelte:2]: Have we looked into using [this](https://kit.svelte.dev/docs/configuration#alias) instead? Seems like it would create both the vite and tsconfig settings, but unsure if those settings are the issue here.

Comment (Alex-Tideman) [reply src/lib/holocene/accordion.svelte:2]: Yeah we have alias config in both ui/cloud-ui.

**Comment by Alex-Tideman** [comment]
So I removed all the $holocene alias references and that fixes the imports. I'm not heartbroken about removing this for now since it's just saving a few keystrokes ($lib/holocene -> $holocene) and the auto-importer usually does this for now. Once we get this in, we can experiment with a single instance of an alias and figure out the right config for it.

File: src/lib/holocene/accordion.svelte
Comment (laurakwhit) [reply src/lib/holocene/accordion.svelte:2]: But in `vite.config.ts` and not [svelte.config.ts](https://kit.svelte.dev/docs/configuration#alias). Was wondering if there was something with the `tsconfig` that could be magically fixed by adding to `svelte.config.ts` instead.

**Pull request merged**
